evimanager/components/lessoncard.js

```diff
@@ -40,7 +41,7 @@ export const LessonCard = ({
         </View>
         <View style={styles.weekendHolidayContainer}>
           <View style={styles.weekendHolidayContentInfoContainer}>
-            <Ionicons name="cafe" size={20} color={'#ffffff'} />
+            <Ionicons name="cafe" size={20} color={THEME.fontColor} />
```

Comment:
Make Icons fit with the Theme (wouldn't be good with a white background aka Light Theme!